src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  totalItems: string[];
```

Comment:
The type for `totalItems` seems to be incorrect. Since `totalItems.length` is used within the component, it implies that `totalItems` should be an array with a length property. The current type `string[]` suggests an array of strings, which might be correct if each item is identified by a string, but if `totalItems` is meant to hold objects representing items, the type should be adjusted accordingly.